tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -474,24 +487,33 @@ def test_search_experiments_filter_by_time_attribute(store: SqlAlchemyStore):
     exp_id2 = store.create_experiment("2")
     exp2 = store.get_experiment(exp_id2)
 
-    experiments = store.search_experiments(filter_string=f"creation_time = {exp1.creation_time}")
+    experiments = store.search_experiments(
+        filter_string=f"creation_time = {exp1.creation_time}"
+    )
     assert [e.experiment_id for e in experiments] == [exp_id1]
 
-    experiments = store.search_experiments(filter_string=f"creation_time != {exp1.creation_time}")
+    experiments = store.search_experiments(
+        filter_string=f"creation_time != {exp1.creation_time}"
+    )
     assert [e.experiment_id for e in experiments] == [
         exp_id2,
         store.DEFAULT_EXPERIMENT_ID,
     ]
 
-    experiments = store.search_experiments(filter_string=f"creation_time >= {time_before_create1}")
+    experiments = store.search_experiments(
+        filter_string=f"creation_time >= {time_before_create1}"
+    )
     assert [e.experiment_id for e in experiments] == [exp_id2, exp_id1]
 
-    experiments = store.search_experiments(filter_string=f"creation_time < {time_before_create2}")
+    experiments = store.search_experiments(
+        filter_string=f"creation_time < {time_before_create2}"
+    )
     assert [e.experiment_id for e in experiments] == [
         exp_id1,
         store.DEFAULT_EXPERIMENT_ID,
     ]
 
+    time.sleep(0.001)
```

Comment:
This is the actual change to ensure `now` below is not the same value as the `exp_id2` create_time.

Reply:
Can we insert a comment?